test/mock-vendor.go

```diff
@@ -0,0 +1,11 @@
+package test
+
+import "github.com/golang/mock/mockgen/model"
+
+// This assignment exists solely for the purpose of convincing godep
+// to vendor github.com/golang/mock/mockgen/model so that we don't
+// require users of the boulder-tools image to have extra packages
```

Comment:
Ideally this comment should explain *why* users would otherwise need this extra package. I.e. "when we run gomock to generate code, it expects to be able to import its model package, and errors if it's not present."